frontend/test/pytest/test_jvpvjp.py

```diff
@@ -56,7 +56,11 @@ def assert_elements_allclose(a, b, **kwargs):
         assert_allclose(i, j, **kwargs)
 
 
-def test_jvp_against_jax_full_argnum_case_S_SS():
+diff_methods = ["auto", "fd"]
```

Comment:
A question: what method do we use if user sets `auto` and there is no `@qnode` around the qjitted function?

Reply:
Classical differentiation always happens with enzyme (backprop) when "auto" is selected.

Reply:
Yes exactly what David said.